recipe/meta.yaml

```
@@ -11,45 +11,59 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<39 or py>=314]
 
 requirements:
   host:
     - pip
-    - python {{ python_min }}
+    - python
     - setuptools
```

https://github.com/mistralai/mistral-common/blob/v1.8.3/pyproject.toml#L2
```suggestion
    - wheel
    - setuptools >=42
```